main.go

```diff
@@ -107,7 +107,7 @@ func realMain() error {
 		flagSkipPrivateFields = flag.Bool("skip-unexported", false, "Skip unexported fields")
 		flagTransform         = flag.String("transform", "snakecase",
 			"Transform adds a transform rule when adding tags."+
-				" Current options: [snakecase, camelcase, lispcase, pascalcase, keep]")
+				" Current options: [snakecase, camelcase, lispcase, pascalcase, keep, titlecase]")
```

Comment:
Can you please add titlecase after `pascalcase`. Let's keep cases tight together.